src/ig/IGExporter.ts

```diff
@@ -513,61 +626,86 @@ export class IGExporter {
               continue;
             }
 
-            const resource: ImplementationGuideDefinitionResource = {
+            const referenceKey = `${resourceJSON.resourceType}/${resourceJSON.id}`;
```

Comment:
This is the third time we're seeing a block of logic like this with a lot of similarities -- but I expect it's one of those cases where things are just slightly-different enough that it can't easily be extracted to a common function?  Assuming that to be the case, we may want to put in a block of comments before each section like this saying the it shares very similar code with two other sections and that when changes are made here, the developer should also refer to the other sections to see if changes need to be made there as well.  What do you think?